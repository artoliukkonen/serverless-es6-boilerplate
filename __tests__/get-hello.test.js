import getHello from '../src/get-healthcheck';

test('Hello World', async () => {
  expect.assertions(1);
  const service = getHello();
  await expect(service).resolves.toMatchSnapshot();
});
