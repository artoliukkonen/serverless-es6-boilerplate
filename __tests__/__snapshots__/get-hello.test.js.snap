// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello World 1`] = `
Object {
  "service": undefined,
  "stage": undefined,
  "version": undefined,
}
`;
